shellbase-core/src/main/scala/com/sumologic/shellbase/ShellBase.scala

```diff
@@ -346,207 +342,25 @@ abstract class ShellBase(val name: String) {
   // Root commands.
   // -----------------------------------------------------------------------------------------------
 
-  val subCommandExtractor = "`([^`]*)`".r // Regex extractor for commands within `-quotes.
+  val subCommandExtractor = ShellBase.SubCommandExtractor
```

Comment:
I left this in to maintain API compatibility since this was a public variable.  I'll strip it out in the future.